### PR TITLE
[strings] Latvian translations from @ttrasuns

### PR DESCRIPTION
**Pull request by biodranik**
@ttrasuns thanks!

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -4525,6 +4624,7 @@
     ja = Organic Maps で私の現在地が見れます！%1$@ または %2$@ オフラインマップはまだインストールしていないですか？ここからダウンロード： https://omaps.app/get
     ko = 현재 위치를 알아 보십시오. %1$@ 또는 %2$@ 열기
     lt = Labas, pažvelk į mano dabartinę vietą Organic Maps! %1$@ arba %2$@ Neturi atjungtinių žemėlapių? Atsisiųsk čia: https://omaps.app/get
+    lv = Apskatiet manu atrašanās vietu „Organic Maps“ lietotnē! %1$@ vai %2$@ Vai Jums nav nesaistes kartes? Lejupielādējiet to: https://omaps.app/get
```
Comment (biodranik) [inline data/strings/strings.txt:4627]: Here Github allows to conveniently review and proof-read any change.

File: data/strings/strings.txt [hunk 2]
```diff
@@ -1033,6 +1054,7 @@
     ja = すべての人のために無料で、愛を込めて作られています
     ko = 사랑으로 만든 모든 사람에게 무료
     lt = Nemokama visiems, sukurta su meile
+    lv = Brīva visiem, veidota ar mīlestību
```
Comment (soshial) [inline data/strings/strings.txt:1057]: Does "free" here means that OM is: 
a) "costs nothing"
b) it's libre software
c) or that's available to everyone?

Comment (biodranik) [reply data/strings/strings.txt:1057]: All of these )

I would focus on the a) and c) meanings.

File: data/strings/strings.txt [hunk 3]
```diff
@@ -3660,6 +3740,7 @@
     ja = トイレ
     ko = 화장실
     lt = Tualetas
+    lv = Tualete
```
Comment (soshial) [inline data/strings/strings.txt:3743]: Maybe "Labierīcības"?

File: data/strings/strings.txt [hunk 4]
```diff
@@ -3937,6 +4023,7 @@
     ja = RV用
     ko = RV용
     lt = Rekreacinio transporto aikštelės
+    lv = Treileru laukumi
```
Comment (soshial) [inline data/strings/strings.txt:4026]: This translation is incorrect, IMHO.
I would say "Labierīcības automājām" or "Labiekārtojums automājām"?

File: data/strings/strings.txt [hunk 5]
```diff
@@ -8755,6 +8948,7 @@
     ja = 位置情報サービスを有効化してください
     ko = 위치 서비스를 작동시켜 주십시요.
     lt = Įjunkite vietovės tarnybas
+    lv = Ieslēdziet ierīces atrašanās vietas servisus
```
Comment (soshial) [inline data/strings/strings.txt:8951]: Hm, varbūt "atrašanās vietas **noteikšanas** servisus/pakalpojumus"?

File: data/strings/strings.txt [hunk 6]
```diff
@@ -10652,6 +10888,7 @@
     ja = 中間地点を調整してください。
     ko = 중간 지점을 조정하세요.
     lt = Suderinkite tarpinį tašką.
+    lv = Precizējiet starppunktu.
```
Comment (soshial) [inline data/strings/strings.txt:10891]: "Pielabojiet starppunktu"?

File: data/strings/strings.txt [hunk 7]
```diff
@@ -10870,6 +11111,7 @@
     ja = マップをダウンロードし、複数マップを利用してより最適なルートを作成しますか？
     ko = 지도를 다운로드하여, 두 개 이상의 지도를 통과하는 더 최적화된 경로를 설정하시겠습니까?
     lt = Norėtumėte atsisiųsti žemėlapį, kad sukurtumėte labiau optimalų maršrutą, apimantį daugiau nei vieną žemėlapį?
+    lv = Vai vēlaties lejupielādēt karti un izvaidot optimālāku maršrutu, kas ietvar vairāk par vienu karti?
```
Comment (soshial) [inline data/strings/strings.txt:11114]: izvaidot -> izveidot
ietvar -> ietver

File: data/strings/strings.txt [hunk 8]
```diff
@@ -30048,6 +30728,7 @@
     ja = ハイキング
     ko = 하이킹
     lt = Žygiai pėsčiomis
+    lv = Pastaigu maršruti
```
Comment (soshial) [inline data/strings/strings.txt:30731]: This is not only "walking routes". Maybe "Aktīvā atpūta" or simply "Daba"?

File: data/strings/strings.txt [hunk 9]
```diff
@@ -29957,6 +30635,7 @@
     it = Concessione dei permessi
     ja = 許可を与える
     ko = 권한 부여
+    lv = Atļaut piekļūt
```
Comment (soshial) [inline data/strings/strings.txt:30638]: "Atļaut piekļuvi"

File: data/strings/strings.txt [hunk 10]
```diff
@@ -29913,6 +30590,7 @@
     it = Questa applicazione richiede l'accesso alla tua posizione per la navigazione.
     ja = このアプリケーションは、ナビゲーションのために位置情報へのアクセスを必要とする。
     ko = 이 애플리케이션을 사용하려면 내비게이션 목적으로 사용자 위치에 액세스해야 합니다.
+    lv = Lai darbotos navigācija, lietotnei ir nepieciešama atrašanās vietas informācija.
```
Comment (soshial) [inline data/strings/strings.txt:30593]: "Lai darbotos navigācija, lietotnei ir nepieciešama **ierīces** atrašanās vieta."

File: data/strings/strings.txt [hunk 11]
```diff
@@ -27357,6 +27976,7 @@
     ja = ガソリンスタンド
     ko = 주유소
     lt = Degalinės
+    lv = DUS
```
Comment (soshial) [inline data/strings/strings.txt:27979]: "Degvielas uzpildes stacijas"

File: data/strings/strings.txt [hunk 12]
```diff
@@ -25747,6 +26330,7 @@
     ja = 携帯電話に切り替えてすべてのブックマークを見る
     ko = 모든 북마크를 보려면 휴대폰으로 전환하세요.
     lt = Perjungimas į telefoną, kad pamatytumėte visas žymes
+    lv = Lai redzētu visas grāmatzīmes, pāŗslēdzieties uz tālruni
```
Comment (soshial) [inline data/strings/strings.txt:26333]: "pāŗslēdzieties" -> "pārslēdzieties"

File: data/strings/strings.txt [hunk 13]
```diff
@@ -24564,6 +25119,7 @@
     ja = 迂回ルートを設定する
     ko = 피할 도로 정의하기
     lt = Apibrėžti vengiamus kelius
+    lv = Definēt nevēlamos ceļus
```
Comment (soshial) [inline data/strings/strings.txt:25122]: "Definēt nevēlamos ceļus" -> "Noteikt nevēlamos ceļus"

File: data/strings/strings.txt [hunk 14]
```diff
@@ -24063,6 +24607,7 @@
     ja = 自動
     ko = 자동
     lt = Automatinis
+    lv = Kad ir zems uzlādes līmenis
     mr = स्वयंचलित
     nb = Automatisk
```
Comment (soshial) [inline data/strings/strings.txt:24612]: I wonder why English translation is different from all other languages? @pastk @biodranik

File: data/strings/strings.txt [hunk 15]
```diff
@@ -22358,6 +22865,7 @@
     it:other = %d file trovati. Li vedrai dopo la conversione.
     ja = %d ファイルが見つかりました。 あなたは変換後にそれらを見るでしょう。
     ko = %d 파일을 찾았습니다. 회심 후 그들을 볼 수 있습니다.
+    lv = Lietotne atrada %d jauna(-as) datne(-es). Varat to (tās) apskatīt pēc konvertācijas.
```
Comment (soshial) [inline data/strings/strings.txt:22868]: lv = Lietotne atrada %d jauna datne. Varat to apskatīt pēc konvertācijas.
lv:other = Lietotne atrada %d jaunas datnes. Varat tās apskatīt pēc konvertācijas.

File: data/strings/strings.txt [hunk 16]
```diff
@@ -20822,6 +21296,7 @@
     ja = 中間地点を追加
     ko = 스톱 추가
     lt = Pridėti sustojimą
+    lv = Pievienot pieturu
```
Comment (soshial) [inline data/strings/strings.txt:21299]: "Pievienot starppunktu".

Because "Starppunkts" is used everywhere else

File: data/strings/strings.txt [hunk 17]
```diff
@@ -20600,6 +21069,7 @@
     ja = 検索を使うか、地図上をタップして目的地を追加する
     ko = 검색을 사용하거나 지도를 탭하여 목적지 지점을 추가합니다
     lt = Naudokite paiešką arba bakstelėkite žemėlapyje, kad pridėtumėte paskirties tašką
+    lv = Lietojiet meklēšanu vai uzsitiet uz kartes, lai pievienotu galamērķi
```
Comment (soshial) [inline data/strings/strings.txt:21072]: "uzsitiet" -> "pieskarieties"

File: data/strings/strings.txt [hunk 18]
```diff
@@ -20554,6 +21022,7 @@
     ja = 検索を使うか、地図上をタップしてルートの出発点を追加する
     ko = 검색을 사용하거나 지도를 탭하여 경로 시작점을 추가합니다
     lt = Naudokite paiešką arba bakstelėkite žemėlapyje, kad pridėtumėte maršruto pradžios tašką
+    lv = Lietojiet meklēšanu vai uzsitiet uz kartes, lai pievienotu sākuma punktu
```
Comment (soshial) [inline data/strings/strings.txt:21025]: "uzsitiet" -> "pieskarieties"

File: data/strings/strings.txt [hunk 19]
```diff
@@ -20240,6 +20701,7 @@
     ja = 一般的なフィードバック
     ko = 일반 피드백
     lt = Bendrasis atsiliepimas
+    lv = Pamata atsauksmes
```
Comment (soshial) [inline data/strings/strings.txt:20704]: "Vispārīgas atsauksmes"

File: data/strings/strings.txt [hunk 20]
```diff
@@ -19889,6 +20342,7 @@
     ja = 常に確認
     ko = 항상 표시
     lt = Visada klausti
+    lv = Vienmēŗ jautāt
```
Comment (soshial) [inline data/strings/strings.txt:20345]: "Vienmēr jautāt"

File: data/strings/strings.txt [hunk 21]
```diff
@@ -17224,6 +17614,7 @@
     ja = 2. 「位置情報」をタップします
     ko = 2. 위치 누르기
     lt = 2. Bakstelėkite „Vietovė“
+    lv = 2. Uzsitiet uz „Vieta“
```
Comment (soshial) [inline data/strings/strings.txt:17617]: Kas ir "uzsitiet"?

File: data/strings/strings.txt [hunk 22]
```diff
@@ -17356,6 +17749,7 @@
     ja = 3.位置情報サービスを選択する
     ko = 3. 3. 위치 서비스 선택
     lt = 3. Pasirinkite Vietos nustatymo paslaugos
+    lv = 3. Atlasieties atrašanās vietas pakalpojumus
```
Comment (soshial) [inline data/strings/strings.txt:17752]: "Atlasieties" -> "Atlasiet"

File: data/strings/strings.txt [hunk 23]
```diff
@@ -16658,6 +17035,7 @@
     ja = 提案した変更は、OpenStreetMapのコミュニティに送信されます。Organic Mapsで編集できない詳細を説明してください。
     ko = 귀하가 제안한 변경 사항은 OpenStreetMap 커뮤니티로 전송됩니다. Organic Maps에서 편집할 수 없는 세부정보를 작성해 주세요.
     lt = Jūsų pasiūlyti žemėlapio pakeitimas bus išsiųsti OpenStreetMap bendruomenei. Aprašykite papildomą informaciją, kuri negali būti redaguojama Organic Maps.
+    lv = Jūsu izmaiņu priekšlikumu nosūtīsim „OpenStreetMap“ kopienai, Lūdzu, sniedziet jebkādu papildu informāciju, ko nevarat norādīt „Organic Maps“ lietotnē.
```
Comment (soshial) [inline data/strings/strings.txt:17038]: "kopienai," -> "kopienai." (dot)

File: data/strings/strings.txt (hunk 3 above)
Comment (richlv) [reply data/strings/strings.txt:3743]: I'd keep "tualete", as "labierīcības" for a toilet seems to be informal.

File: data/strings/strings.txt (hunk 7 above)
Comment (richlv) [reply data/strings/strings.txt:11114]: "Optimālāku" is an incorrect form - suggesting "piemērotāku" ("optimāls" already is the highest possible level, cannot gain "-āks").

File: data/strings/strings.txt (hunk 8 above)
Comment (richlv) [reply data/strings/strings.txt:30731]: "Daba" could be confusing (is this trees, rivers etc?), "aktīvā atpūta" seems reasonable.

File: data/strings/strings.txt (hunk 15 above)
Comment (richlv) [reply data/strings/strings.txt:22868]: Latvian uses singular for all numbers, ending with 1, except 11.
Can this be expressed in the localisation framework here?

File: data/strings/strings.txt (hunk 17 above)
Comment (richlv) [reply data/strings/strings.txt:21072]: And then also "uz kartes" -> "kartei".

File: data/strings/strings.txt (hunk 18 above)
Comment (richlv) [reply data/strings/strings.txt:21025]: And then also "uz kartes" -> "kartei".

File: data/strings/strings.txt [hunk 24]
```diff
@@ -20328,6 +20791,7 @@
...
     ko = 일부 언어의 경우 앱 스토어(Google Play, Galaxy Store, App Gallery, FDroid)에서 다른 음성 합성기 또는 추가 언어 팩을 설치해야 합니다.\n장치의 설정 → 언어 및 입력 → 음성 → 텍스트-음성 변환 출력을 엽니다.\n여기서 음성 합성에 대한 설정을 관리하고(예: 오프라인 사용을 위한 언어 팩 다운로드) 다른 텍스트-음성 변환 엔진을 선택할 수 있습니다.
     lt = Kai kurioms kalboms jums reikės įdiegti šnekos sintezatorių arba papildomą kalbos paketą iš programėlių parduotuvės (Play parduotuvė, Galaxy parduotuvė, Programėlių galerija, FDroid).\nAtidarykite įrenginio nustatymus → Kalbos ir įvestis → Kalba → Teksto į kalbą išvestis.\nČia galite tvarkyti nustatymus šnekos sintezatoriui (pavyzdžiui, atsisiųsti kalbos paketą) ir pasirinkti kitą teksto į kalbą variklį.
+    lv = Dažām valodām nepieciešams instalēt runas sintezatoru vai papildu valodas paku, kas parasti atrodama lietotņu veikalā („Google Play“, „Galaxy Store“, „App Gallery“, „FDroid“).\nAtveriet ierīces iestatījumus → Valoda un ievade → Runa → Teksta pārveide runā.\nTe varat pārvaldīt runas sintēzes iestatījumus (piemēram, lejupielādēt valodas paku nesaistes lietošanai), kā arī varat nomainīt teksta pārvēršanas runā dzini.
```
[the remark this answers is not in the document]
Comment (richlv) [reply data/strings/strings.txt:20794]: Maybe "varat nomainīt teksta pārvēršanas runā dzini" -> "varat nomainīt dzini teksta pārvēršanai runā"?
The current form sort of surprises the reader with each word. The suggested one is not ideal, but it seems to allow easier parsing.

File: data/strings/strings.txt (hunk 15 above)
Comment (soshial) [reply data/strings/strings.txt:22868]: Of course, [Unicode standard](https://www.unicode.org/cldr/charts/45/supplemental/language_plural_rules.html#lv) describes it in details and it is definitely supported in Android (not sure about other OSes). I will edit this right now @richlv

File: data/strings/strings.txt (hunk 2 above)
Comment (soshial) [reply data/strings/strings.txt:1057]: "Bezmaksas un pieejama visiem, un veidota ar mīlestību"

File: data/strings/strings.txt (hunk 8 above)
Comment (soshial) [reply data/strings/strings.txt:30731]: Maybe "Brīvdabas" / "Brīvdabā"?

File: data/strings/strings.txt (hunk 2 above)
Comment (richlv) [reply data/strings/strings.txt:1057]: "Bezmaksas, pieejama visiem un veidota ar mīlestību"? :)

File: data/strings/strings.txt (hunk 1 above)
Comment (soshial) [reply data/strings/strings.txt:4627]: We need @ttrasuns' opinion on our proposed changes.

File: data/strings/strings.txt [hunk 25]
```diff
@@ -21424,6 +21911,7 @@
     it:other = %d preferiti
     ja = %d個のブックマーク
     ko = %d 북마크
+    lv = Grāmatzīmes: %d
```
Comment (soshial) [inline data/strings/strings.txt:21914]: lv:zero = %d grāmatzīmju
lv:one = %d grāmatzīme
lv:other = %d grāmatzīmes

File: data/strings/strings.txt [hunk 26]
```diff
@@ -22615,6 +23127,7 @@
     it:other = %d percorsi
     ja = %dの追跡
     ko = %d 추적
+    lv = Ceļi: %d
```
Comment (soshial) [inline data/strings/strings.txt:23130]: lv:zero = %d ceļu
lv:one = %d ceļš
lv:other = %d ceļi

File: data/strings/strings.txt [hunk 27]
```diff
@@ -11748,6 +12009,7 @@
     ja = 目的地
     ko = 목적지
     lt = Maršrutas į
+    lv = Maršruts uz
```
Comment (soshial) [inline data/strings/strings.txt:12012]: This might be too long for a small button. Maybe "Ceļš uz" and "Ceļš no"?

**Comment by soshial** [comment]
I've just written to @ttrasuns and asked to participate and provide feedback on my corrections — let's wait for their response. I would propose to merge this PR as soon as possible to avoid divergence of the texts.

File: data/strings/strings.txt (hunk 4 above)
Comment (ttrasuns) [reply data/strings/strings.txt:4026]: I think the best version would be "Atpūtas transportlīdzekļu labierīcības".

RVs consists of motorhomes and trailers. Yes, "treileru laukumi" is a mistake as it does not encompass motorhomes.
For motorhomes there are no approved terms, I can find both "motormāja" and "automāja" used. However, is does not encompass whole RVs either.
I can't find anything for RV in dictionaries, but "atpūtas transportlīdzeklis" is a term used in EU documentation among "atpūtas un izklaides transportlīdzklis", "transportlīdzeklis izklaidei" etc.

File: data/strings/strings.txt (hunk 5 above)
Comment (ttrasuns) [reply data/strings/strings.txt:8951]: I agree "Ieslēdziet ierīces atrašanās vietas noteikšanas pakalpojumus".

File: data/strings/strings.txt (hunk 6 above)
Comment (ttrasuns) [reply data/strings/strings.txt:10891]: Then, I think, it stylistically would be better to write "Labojiet starppunktu." without the prefix.
But I'm not sure at what circumstances this message shows up?

File: data/strings/strings.txt (hunk 7 above)
Comment (ttrasuns) [reply data/strings/strings.txt:11114]: Yes, so with both corrections — "Vai vēlaties lejupielādēt karti un izveidot piemērotāku maršrutu, kas ietvar vairāk par vienu karti?"

File: data/strings/strings.txt (hunk 8 above)
Comment (ttrasuns) [reply data/strings/strings.txt:30731]: I think we can use "Aktīvā atpūta" to encompass everything.

File: data/strings/strings.txt (hunk 9 above)
Comment (ttrasuns) [reply data/strings/strings.txt:30638]: Yes, "Atļaut piekļuvi"

File: data/strings/strings.txt (hunk 10 above)
Comment (ttrasuns) [reply data/strings/strings.txt:30593]: I think "ierīces atrašanās vieta" sounds confusing (at least for me), as a device always have a location. Or is there any other reason to change it (length etc.)?

Or maybe you meant "**Lai darbotos navigācija, lietotnei ir nepieciešama ierīces atrašanās vietas informācija.**" to which I can agree.

File: data/strings/strings.txt (hunk 11 above)
Comment (ttrasuns) [reply data/strings/strings.txt:27979]: Yes, if length is not an issue, I agree to change.

File: data/strings/strings.txt (hunk 12 above)
Comment (ttrasuns) [reply data/strings/strings.txt:26333]: Yes, thanks for spotting!

File: data/strings/strings.txt (hunk 13 above)
Comment (ttrasuns) [reply data/strings/strings.txt:25122]: Yes, "Noteikt nevēlamos ceļus" sounds much better!

File: data/strings/strings.txt (hunk 14 above)
Comment (ttrasuns) [reply data/strings/strings.txt:24612]: I can't comment on this one.

File: data/strings/strings.txt (hunk 15 above)
Comment (ttrasuns) [reply data/strings/strings.txt:22868]: Is Android using Unicode or GNU gettext standard for plurals? Or maybe doesn't matter in this case?

A correction in lv:one

lv:zero = Lietotne atrada %d jaunu datņu. Varat tās apskatīt pēc konvertācijas.
lv:one = Lietotne atrada %d jaunu datni. Varat to apskatīt pēc konvertācijas.
lv:other = Lietotne atrada %d jaunas datnes. Varat tās apskatīt pēc konvertācijas.

File: data/strings/strings.txt (hunk 16 above)
Comment (ttrasuns) [reply data/strings/strings.txt:21299]: I agree to the change. (but that means Lithuanian is wrong as well).

File: data/strings/strings.txt (hunk 17 above)
Comment (ttrasuns) [reply data/strings/strings.txt:21072]: Yes, and thanks —
"Lietojiet meklēšanu vai pieskarieties kartei, lai pievienotu galamērķi"

File: data/strings/strings.txt (hunk 18 above)
Comment (ttrasuns) [reply data/strings/strings.txt:21025]: Thanks —
"Lietojiet meklēšanu vai pieskarieties kartei, lai pievienotu sākuma punktu"

File: data/strings/strings.txt (hunk 24 above)
[the remark this answers is not in the document]
Comment (ttrasuns) [reply data/strings/strings.txt:20794]: I think "uzstādīt" is a common mistake. "Instalēt" is correct one. See akadterm.lv etc

I agree to @richlv 

Dažām valodām nepieciešams instalēt runas sintezatoru vai papildu valodas paku, kas parasti atrodama lietotņu veikalā („Google Play“, „Galaxy Store“, „App Gallery“, „FDroid“).\nAtveriet ierīces iestatījumus → Valoda un ievade → Runa → Teksta pārveide runā.\nTe varat pārvaldīt runas sintēzes iestatījumus (piemēram, lejupielādēt valodas paku lietošanai nesaistē, kā arī varat nomainīt dzini teksta pārvēršanai runā.

File: data/strings/strings.txt (hunk 19 above)
Comment (ttrasuns) [reply data/strings/strings.txt:20704]: I agree —
**"Vispārīgas atsauksmes"**

File: data/strings/strings.txt (hunk 20 above)
Comment (ttrasuns) [reply data/strings/strings.txt:20345]: Thanks, yes —
"Jautāt vienmēr"

File: data/strings/strings.txt (hunk 21 above)
Comment (ttrasuns) [reply data/strings/strings.txt:17617]: Thanks —
"Pieskarieties pie „Vieta“"

File: data/strings/strings.txt (hunk 22 above)
Comment (ttrasuns) [reply data/strings/strings.txt:17752]: Yes, thanks —
"3. Atlasiet atrašanās vietas pakalpojumus"

File: data/strings/strings.txt (hunk 23 above)
Comment (ttrasuns) [reply data/strings/strings.txt:17038]: Yes, a dot is needed:
"Jūsu izmaiņu priekšlikumu nosūtīsim „OpenStreetMap“ kopienai. Lūdzu, sniedziet jebkādu papildu informāciju, ko nevarat norādīt „Organic Maps“ lietotnē."

File: data/strings/strings.txt (hunk 25 above)
Comment (ttrasuns) [reply data/strings/strings.txt:21914]: Thanks, perfect!

File: data/strings/strings.txt (hunk 26 above)
Comment (ttrasuns) [reply data/strings/strings.txt:23130]: Thanks!

File: data/strings/strings.txt (hunk 27 above)
Comment (ttrasuns) [reply data/strings/strings.txt:12012]: Thanks, I like "Kurp", it sounds fresh!

File: data/strings/strings.txt (hunk 7 above)
Comment (soshial) [reply data/strings/strings.txt:11114]: You made this mistake again:
ietvar -> ietver

File: data/strings/strings.txt (hunk 4 above)
Comment (soshial) [reply data/strings/strings.txt:4026]: Manā tehniskajā pasē ir rakstīts "Vieglais dzīvojamais" (transportlīdzeklis).

So what do you think about this?
"Dzīvojamo auto labierīcības"

File: data/strings/strings.txt (hunk 6 above)
Comment (soshial) [reply data/strings/strings.txt:10891]: Navigation cannot build route with the chosen "intermediate point" and asks to fix the route by moving it somewhere else. "Please adjust your intermediate point." isn't the best text that can be written in English either. @biodranik @rtsisyk ideas?

photo_5436154095711940181_y

File: data/strings/strings.txt (hunk 4 above)
Comment (ttrasuns) [reply data/strings/strings.txt:4026]: Paldies par informāciju! Man nav bijusi tuvāka pazīšanās ar treileriem un motormājām.

„Dzīvojamo auto labierīcības“ šķiet adekvāti un varbūt tad arī paliekam pie tā, jo izklausās, ka tāda man 100% skaidra varianta nav.

File: data/strings/strings.txt (hunk 6 above)
Comment (ttrasuns) [reply data/strings/strings.txt:10891]: It is a pretty generic comment text, so it functions to encourage the user to do something without specifying the problem. I would guess  that "Labojiet starppunktu" or "Pārbaudiet starppunktu" both work.

File: data/strings/strings.txt (hunk 7 above)
Comment (ttrasuns) [reply data/strings/strings.txt:11114]: Sorry, tired brain, it should be correct now — "Vai vēlaties lejupielādēt karti un izveidot piemērotāku maršrutu, kas ietver vairāk par vienu karti?"

**Comment by soshial** [comment]
@ttrasuns has provided feedback to all my comments and all were resolved. May I ask @biodranik to make corresponding edits to this PR?

**Comment by ttrasuns** [comment]
This is the end file for Latvian translation:
https://gist.github.com/ttrasuns/41224b8304c2ba7d0b510996e9f51423

**Comment by ttrasuns** [comment]
Here is categories.txt translated in Latvian.
https://gist.github.com/ttrasuns/847ea00d98e3947a9477746622608135

**Comment by biodranik** [comment]
Thanks! Please add new/missing translations for:
- editor_other_info
- editor_note_hint
- editor_about_osm

Made a separate PR for categories: https://github.com/organicmaps/organicmaps/pull/9486

**Comment by soshial** [comment]
editor_other_info
```
en = Note to OpenStreetMap volunteers (optional)
lv = Piezīme OpenStreetMap brīvprātīgajiem (pēc izvēles)
```

editor_note_hint
```
en = Describe errors on the map or things that cannot be edited with Organic Maps
lv = Aprakstiet kļūdas kartē vai vienumus, kurus nevar labot Organic Maps lietotnē
```

editor_about_osm
```
en = Your edits are uploaded to the public <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> database. Please do not add personal or copyrighted information.
lv = Jūsu labojumi tiks augšupielādēti publiskajā <a href="https://wiki.openstreetmap.org/wiki/About_OpenStreetMap">OpenStreetMap</a> datubāzē. Lūdzu, nepievienojiet personīgu vai ar autortiesībām aizsargātu informāciju.
```

@biodranik

**Review by soshial — APPROVED** [review]

**Review by vng — APPROVED** [review]

**Comment by vng** [comment]
Anyway, lv language is still missing when generate_localization (same as "lt"), should add stubs?

**Comment by soshial** [comment]
@vng  could you please explain what you mean? AFAIU, generate_localizations.sh was executed and "lv" was added to the locale list

File: data/strings/strings.txt (hunk 16 above)
Comment (soshial) [reply data/strings/strings.txt]: Dear brother Lithuanians, @organicmaps/translations-lt , are you sure that the word stop (as in "bus stop") is the best choice in "Pridėti sustojimą"? In Latvian we translated as smth like "middle-location".

**Comment by pastk** [comment]
It makes sense to release lv translations altogether with https://github.com/organicmaps/organicmaps/pull/9486

**Pull request merged**